### ISSUE TITLE
linetype="curve" option not working

### ISSUE THREAD

**Issue by juliohm** [JuliaGraphs/GraphPlot.jl#6]
``` julia
using LightGraphs
using GraphPlot

G = DiGraph(2)
add_edge!(G, 1,2)
add_edge!(G, 2,1)

display(gplot(G, linetype="curve"))
```

> LoadError: MethodError: `convert` has no method matching convert(::Type{Measures.Measure}, ::Float64)
> This may have arisen from a call to the constructor Measures.Measure(...),
> since type constructors fall back to convert methods.
> Closest candidates are:
>   call{T}(::Type{T}, ::Any)
>   convert{T}(::Type{T}, !Matched::T)
> while loading In[50], in expression starting on line 8
> 
> in call at /home/julio/.julia/v0.5/Compose/src/form.jl:894
>  in parsepathop at /home/julio/.julia/v0.5/Compose/src/form.jl:901
>  in parsepath at /home/julio/.julia/v0.5/Compose/src/form.jl:1098
>  in path at /home/julio/.julia/v0.5/Compose/src/form.jl:1125
>  in **gplot#7** at /home/julio/.julia/v0.5/GraphPlot/src/plot.jl:191
>  in gplot at /home/julio/.julia/v0.5/GraphPlot/src/plot.jl:217
>  [inlined code] from essentials.jl:114

**Comment by juliohm** [comment]
This issue can be tracked down to this line in the source: https://github.com/afternone/GraphPlot.jl/blob/master/src/plot.jl#L191

**Comment by juliohm** [comment]
@afternone , this is currently the main issue I have. I am trying to debug this algorithm for `LightGraphs.jl` and the labels of antiparallel edges are overwritten. Do you know why this error is occurring? I am assuming that with curved edges the labels won't be placed in the same location.

**Comment by afternone** [comment]
Curve line type is bit complicated. We can solve the overwritten problem in an easy way as shown in #4.

**Comment by juliohm** [comment]
@afternone I would say that curvy edges with labels halfway between nodes look much nicer than the solution in #4. :) Anyways, I can debug my algorithms now that you fixed that other issue, thank you! This package is being really helpful :+1:

**Comment by simonschoelly** [comment]
Finally fixed by #90
